ember-cli-build.js

```diff
@@ -8,13 +8,5 @@ module.exports = function(defaults) {
     // Add options here
   });
 
-  // add the StripTestSelectorsTransform to the registry, so the dummy app has
-  // it added in the HTMLBars build pipeline: all data-test-* attributes are
-  // therefore stripped
-  app.registry.add('htmlbars-ast-plugin', {
-    name: 'strip-test-selectors',
-    plugin: StripTestSelectorsTransform
-  });
-
```

Comment:
Currently the tests assume that the `[data-test-*]` attributes are stripped. Removing the plugin here won't strip them anymore in the tests and that's why they are failing...